test/e2e/config_test.go

```diff
@@ -351,15 +353,18 @@ data:
 			"values": asYAML(t, map[string]interface{}{
 				"existing": func() interface{} {
 					raw := cm.Raw()
+					removeHistory(raw)
```

Comment:
Consequence of https://github.com/vmware-tanzu/carvel-kapp/pull/392/files#r768970039  is that rebase rules don't see history in objects passed to them. I've adjusted test to reflect it.